conftest.py

```diff
@@ -295,24 +295,19 @@ def data_repository(directory_tree):
 
 
 @pytest.fixture(
-    params=[
-        {
-            'name': 'old-datasets-v0.3.0.git',
-            'exit_code': 1
-        },
-        {
-            'name': 'old-datasets-v0.5.0.git',
-            'exit_code': 1
-        },
-        {
-            'name': 'old-datasets-v0.5.1.git',
-            'exit_code': 0
-        },
-        {
-            'name': 'test-renku-v0.3.0.git',
-            'exit_code': 1
-        },
-    ],
+    params=[{
```

Comment:
Which version of yapf are you using?

Reply:
0.27.0. I'll update to 0.30.0

Reply:
Htough 0.27.0 is the one we use, so the question is more, what version of yapf are you using? :smile: 

0.30.0 does indeed reformat some stuff, 0.27.0 leaves the code as is

https://github.com/SwissDataScienceCenter/renku-python/blob/master/setup.py#L114